core/src/main/scala/za/co/absa/spline/core/harvester/operationBuilders.scala

```diff
@@ -124,6 +124,52 @@ abstract class WriteNodeBuilder
     writeMetrics = writeMetrics,
     readMetrics = readMetrics
   )
+
+  override def ignoreLineageWrite:Boolean = {
+    writeMetrics.get("numFiles").filter(0.>).isDefined
```

Comment:
why < 0? Is it correct?
When write was ignored the `numFiles` are set to explicit 0, not a negative value.

Reply:
Looks like bug :(